caddyhttp/httpserver/server.go

```diff
@@ -510,6 +510,17 @@ func (s *Server) OnStartupComplete() {
 	if caddy.Quiet {
 		return
 	}
+	s.OutputSiteInfo()
+}
+
+func (s *Server) OutputSiteInfo() {
+	firstSite := s.sites[0]
+	scheme := "HTTP"
+	if firstSite.TLS.Enabled {
+		scheme = "HTTPS"
+	}
+	fmt.Println("")
+	fmt.Printf("Serving %s on port "+firstSite.Port()+" \n", scheme)
```

Comment:
Could this not be like this?
```suggestion
	fmt.Println("Serving %s on port "+firstSite.Port(), scheme)
```

Reply:
Nevermind, it can't.